src/asmcnc/core_UI/new_popups/job_validation_popup.py

```diff
@@ -0,0 +1,38 @@
+from kivy.base import runTouchApp
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.button import Button
+from kivy.uix.label import Label
+
+from asmcnc.comms.localization import Localization
+from asmcnc.core_UI import scaling_utils
+from asmcnc.core_UI.new_popups.popup_bases import PopupBase, PopupErrorTitle, PopupScrollableBody
+
+
+class JobValidationPopup(PopupBase):
+    def __init__(self, text, **kwargs):
+        super(JobValidationPopup, self).__init__(**kwargs)
+        localisation = Localization()
+
+        title = PopupErrorTitle(size_hint_y=0.15, localisation=localisation)
```

Comment:
Why not use localization as singleton directly in the class and not forwarding it?

Reply:
It is always instantiated from a class which will have localisation. Acquiring the lock is an expensive operation so I thought this makes sense.

Reply:
But if you think otherwise happy to change it 😁

Reply:
👍